app/controllers/manage_cmr/orders_controller.rb

```diff
@@ -24,7 +57,48 @@ def show
     render :index
   end
 
-  def search
+  def show_order
+    response = cmr_client.get_order(token, params['id'])
+    if response.errors
+      Rails.logger.error("#{request.uuid} - OrdersController#show_order - Search Order Error: #{response.clean_inspect}")
+      err_message = { error: "#{response.errors}  Please refer to the ID: #{request.uuid} when contacting #{view_context.mail_to('support@earthdata.nasa.gov', 'Earthdata Support')}" }
```

Comment:
I see the change, that looks great. As for the question of "Earthdata Support" also being moved to config, out of the two, email address and label, the email address has actually changed in the past, so that is why I suggested it as a config, the other value is your call.